money-api/src/main/java/com/comcast/money/api/MoneyTracer.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2012 Comcast Cable Communications Management, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.comcast.money.api;
+
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.trace.Tracer;
+
+public interface MoneyTracer extends Tracer {
```

Comment:
Is this for Java compatibility?

Reply:
It's for covariant returns, I wanted to be able to return the Money flavor of `Span` instead of the OTel `Span`, that way consumers wouldn't have to cast.  Doesn't seem like Scala supports covariant returns directly but Java lets me override the members in an interface that extends from the original interface.